nicegui/elements/knob.py

```diff
@@ -0,0 +1,42 @@
+from ..dependencies import register_component
+from ..element import Element
+
+register_component("knob", __file__, "knob.js")
+
+
+class Knob(Element):
```

Comment:
I would love to have `Knob` deriving from `ValueElement`. This way we could read, write and bind the current value. See `progress.py` for examples.